### ISSUE TITLE
support dot at the end of website domain name

### ISSUE THREAD

**Issue by ilgooz** [mesg-foundation/mesg.com#8]
currently https://mesg.com. responses with:

![image](https://user-images.githubusercontent.com/6023117/43012219-7c0e8012-8c4e-11e8-9cd4-9b1d185e4f5a.png)

**Comment by antho1404** [comment]
I don't really see the problem with that, I never saw a website with a `.` in the end, i'm actually surprised that chrome recognize this as a url.

What are the advantages to support a `.` in the end ?

**Comment by ilgooz** [comment]
This not a major issue but dot at the of domain names is just a part of FQDN spec. Generally domain names ending with a dot will be redirected to the same url without a dot. This could be related with load balancer configs. I guess, I came across with this when I clicked a link like this [http://mesg.com.](http://mesg.com.) in the documentation or repo issues somewhere.

![image](https://user-images.githubusercontent.com/6023117/43034936-747d1070-8cee-11e8-949f-23dec1d02a84.png)
